packages/stratimikos/adapters/CMakeLists.txt

```diff
@@ -36,4 +36,7 @@ ENDIF()
 ASSERT_DEFINED(${PACKAGE_NAME}_ENABLE_Belos)
 IF (${PACKAGE_NAME}_ENABLE_Belos)
   ADD_SUBDIRECTORY(belos)
+  IF (${PACKAGE_NAME}_ENABLE_ThyraTpetraAdapters)
+    ADD_SuBDIRECTORY(belos/tpetra)
```

Comment:
I know CMake is resilient to upper/lowercase, but it definitely bothers my OCD side :stuck_out_tongue: